src/backend/distributed/metadata/node_metadata.c

```diff
@@ -868,6 +858,22 @@ ActivateNode(char *nodeName, int nodePort)
 		ereport(ERROR, (errmsg("node at \"%s:%u\" does not exist", nodeName, nodePort)));
 	}
 
+	/*
+	 * Delete existing reference and replicated table placements on the
+	 * given groupId if the group has been disabled earlier (e.g., isActive
+	 * set to false).
+	 *
+	 * Sync the metadata to all existing metadata nodes irrespective of the
```

Comment:
When I read 'sync the metadata' I expect to see 'SyncMetadataSnapshotToNode' call (or sth similar). Can we update the comment accordingly?